src/App.js

```diff
@@ -26,7 +31,26 @@ export default function App() {
       }}
     >
       <Header>Viary</Header>
-      <MemoriesPage memories={listOfMemories} />
+      {currentPage === 'memories' && (
```

Comment:
What happend to react router?

Reply:
I tried to make the back-button/ page-change with the react router. It felt strange to put it in the DetailMemoryPage and it only worked with a reload, so I changed the way. I read the using in undersides is possible, could this be another way? (Later I will use it anyway).